.github/workflows/pull_request.yml

```diff
@@ -33,13 +33,16 @@ jobs:
       # :-( https://github.community/t/support-for-yaml-anchors/16128
       - {uses: actions/checkout@v2, with: {fetch-depth: 0}}
       - uses: actions/setup-python@v2
-      - name: temporary python3-pil HACK because github is out of date
-        run: sudo apt-get -y install libimagequant0 libwebpdemux2 &&
-           wget http://security.ubuntu.com/ubuntu/ubuntu/pool/main/p/pillow/python3-pil_7.0.0-4ubuntu0.2_amd64.deb &&
-           sudo dpkg -i python3-pil_*.deb
+
+      # Package index is "out of date by design"
+      # https://github.com/actions/virtual-environments/issues/1757
+      - name: apt-get -y update
+        run: sudo apt-get -y update
+
       - name: get python libs
         # FIXME: apt-get succeeds but 'import numpy' still fails!?
```

Comment:
We still have failure with "import numpy" after installing python3-numpy?
I installed with pip3 but it should be the same, right?

Reply:
What happened here is `pylint` complaining that it cannot find `python3-numpy`. Maybe it was specific to pylint and maybe it was specific to github and maybe it's not happening anymore. Anyway it's off-topic in this PR :-)